buildAll.sh

```diff
@@ -114,7 +123,7 @@ is_build_link() {
 }
 is_kernel_file() {
     local str="$1"
-    [[ $str =~ \>(openliberty-)([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)-(.*)\.zip ]]
+    [[ $str =~ \>(openliberty-kernel-)([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)-(.*)\.zip ]]
```

Comment:
The approach to get kernal binary seems to be fine. But http://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/nightly/2020-11-10_1100/ doesn't seem to have such binary though.